ui/admin/package.json

```diff
@@ -37,7 +37,8 @@
   },
   "dependencies": {
     "ember-named-blocks-polyfill": "^0.2.5",
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
+    "uuid": "^11.0.3"
```

Comment:
Interesting enough, I add `uuid` as dependency as a safe guard, but without adding it as dep, the test above still working. I think, it works because uuid is already within `node_modules` due to being a dependency on desktop.

Let me knoe your thoughts on this

Reply:
Yeah we share the same dependencies/node_modules since they're part of the overall monorepo (except for the electron app).